client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-preview-pane/site-preview-pane-header/style.scss

```diff
@@ -1,59 +1,108 @@
+			background-size: cover;
+			background-repeat: no-repeat;
+			height: 100px;
+		}
+
+		.site-preview__header-favicon {
+			position: relative;
+			top: -44px;
+			left: 48px;
+			width: 64px;
+			height: 64px;
+			margin-bottom: -64px;
+		}
+
+		.site-preview__header-content {
+			padding: 48px 48px 24px 48px;
+			display: flex;
+			align-items: center;
 
-	.site-preview__header-favicon {
-		position: relative;
-		top: -44px;
-		left: 48px;
-		width: 64px;
-		height: 64px;
-		margin-bottom: -64px;
+			.site-preview__header-title-summary {
+				flex-grow: 1;
+				display: flex;
+				flex-direction: column;
+
+				.site-preview__header-title {
+					font-style: normal;
+					font-weight: 500;
+					font-size: rem(32px);
+					line-height: 32px;
+					color: var(--studio-black);
+					margin-bottom: 4px;
+				}
+
+				.site-preview__header-summary {
+
+					.site-preview__header-summary-link {
+						display: flex;
+						align-items: center;
+						font-size: rem(18px);
+						font-weight: 400;
+						color: var(--studio-gray-70);
+						line-height: 26px;
+						width: fit-content;
+
+						&:focus {
+							box-shadow: none;
+						}
+					}
+				}
+			}
+		}
 	}
 
-	.site-preview__header-content {
-		padding: 48px 48px 24px 48px;
+	@media (max-width: 960px) {
```

Comment:
We should use the standard media queries-breakpoints: https://wpcalypso.wordpress.com/devdocs/docs/coding-guidelines/css.md#media-queries

- break-mobile
- break-large
- ...

Here are the breakpoints: https://github.com/WordPress/gutenberg/blob/trunk/packages/base-styles/_breakpoints.scss